### PR TITLE
fix: git spinner

### PR DESCRIPTION
**Pull request by c-ehrlich**
## ✅ Checklist

- [x] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-10-06).
- [x] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I performed a functional test on my final commit

---

## Changelog

hopefully fixes the git spinner? leaving as draft for now

💯

### CODE REVIEW

File: cli/src/helpers/git.ts [hunk 1]
```diff
@@ -96,6 +99,7 @@ export const initializeGit = async (projectDir: string) => {
 
   // We're good to go, initializing the git repo
   try {
+    spinner.stop();
```
Comment (juliusmarminge) [inline cli/src/helpers/git.ts:102]: should this be here? @c-ehrlich

Comment (c-ehrlich) [reply cli/src/helpers/git.ts:102]: probably not, i think its leftover from trying stuff

Comment (c-ehrlich) [reply cli/src/helpers/git.ts]: removed this in [5c95429](https://github.com/t3-oss/create-t3-app/pull/621/commits/5c95429f148b5f1b5f8ef7c59b9b8078997a49b9)

**Comment by c-ehrlich** [comment]
I've left the `stop()` instead of `stopAndPersist()` for now.

Reference:
stopAndPersist (old) on the left
stop (new) on the right

stopAndPersist vs stop

can change it back of course

**Review by juliusmarminge — APPROVED** [review]
Ship it

**Pull request merged**